files/zh-cn/web/api/notifications_api/using_the_notifications_api/index.html

```diff
@@ -284,7 +284,7 @@ <h2 id="规范">规范</h2>
 
 <h2 id="浏览器兼容性">浏览器兼容性</h2>
 
-<p>{{page("/zh-CN/Web/API/Notification",".E6.B5.8F.E8.A7.88.E5.99.A8.E5.85.BC.E5.AE.B9.E6.80.A7")}}</p>
+<p>{{page("/en-US/Web/API/Notification","Browser compatibility")}}</p>
```

Comment:
Please replace with this macro `{{Compat("api.Notification")}}`

Reply:
Fixed!